### PR TITLE
feat: add slowsort algorithm

### PR DESCRIPTION
**Pull request by tiang**

### CODE REVIEW

File: src/space_flight/pipelines/data_engineering/nodes.py [hunk 1]
```diff
@@ -101,3 +101,37 @@ def create_master_table(
     master_table = with_companies.drop(["shuttle_id", "company_id"], axis=1)
     master_table = master_table.dropna()
     return master_table
+
+
+# Implementation of a sorting algorithm
+# Source: https://bosker.wordpress.com/2011/04/29/the-worst-algorithm-in-the-world/
+def _slowsort(values, i, j):
```
Comment (george-mathews) [inline src/space_flight/pipelines/data_engineering/nodes.py:108]: Bad implementation - let me know when you have fixed it!!!!!! 
image

Comment (tiang) [reply src/space_flight/pipelines/data_engineering/nodes.py:108]: i'll reply too

Comment (george-mathews) [reply src/space_flight/pipelines/data_engineering/nodes.py:108]: Ready?

Comment (tiang) [reply src/space_flight/pipelines/data_engineering/nodes.py:108]: another comment

Comment (george-mathews) [reply src/space_flight/pipelines/data_engineering/nodes.py:108]: This is pure !@#$!@#$!@$. My dog writes better code than this.   
image

Comment (tiang) [reply src/space_flight/pipelines/data_engineering/nodes.py:108]: does it tho?

Comment (tiang) [reply src/space_flight/pipelines/data_engineering/nodes.py:108]: That's !@#$!@#$!@$. This is a great algorithm.  Also   
image

Comment (george-mathews) [inline src/space_flight/pipelines/data_engineering/nodes.py:108]: If you're implementing a sort feature, looping over the keywords 100 times might be inefficient due to x, maybe we could consider using the borat_sort algorithm instead.


What do you think?

Comment (tiang) [reply src/space_flight/pipelines/data_engineering/nodes.py:108]: Thanks for the feedback, let's discuss offline, not sure I understand how to implement this but would love your thoughts.

Comment (george-mathews) [reply src/space_flight/pipelines/data_engineering/nodes.py:108]: No worries, I've found this implementation on stack overflow that I think will be relevant 

```python
import random
def is_sorted(data) -> bool:
    """Determine whether the data is sorted."""
    return all(data[i] <= data[i+1] for i in range(len(data)-1))


def borat_sort(data) -> list:
    """Shuffle data until sorted."""
    while not is_sorted(data):
        random.shuffle(data)
    return dat
```

Comment (tiang) [reply src/space_flight/pipelines/data_engineering/nodes.py:108]: image

**Pull request closed without merging**